org-kanban.el

```diff
@@ -103,14 +103,40 @@
      (define-key map org-kanban/next-key (lambda () (interactive) (org-kanban/shift 'right)))
      map)))
 
-(defun org-kanban/initialize ()
+(defun org-kanban/initialize (&optional arg)
   "Create an org-kanban dynamic block"
+  (interactive "p")
+  (message "%s" arg)
+  (cond ((eq arg nil) (org-kanban/initialize-here))
+        ((eq arg 1) (org-kanban/initialize-here))
```

Comment:
for which case is this needed?

Reply:
ah .. i played  around with it ...
interactive is evil :)
"p" vs. "P" ...  i think your implementation is quite complete ...